docs/documentation/sbom-guide.md

```diff
@@ -2,425 +2,239 @@
 
-###### Last updated: 2023-02-03
-
-&nbsp;
+###### Last updated: 2024-09-16
 
 # AlmaLinux SBOM User Guide 
 
-&nbsp;
-
-## Introduction
-
-### About *SBOM*
+## About *SBOM*
 
 *SBOM (Software Bill of Materials)* is a list of all open-source and third-party components used in a codebase, along with their licensing information, version numbers, and any known vulnerabilities. 
 
-[AlmaLinux Build System](https://github.com/AlmaLinux/build-system) has implemented *SBOM* into its pipeline for security purposes, such as tracing the build process, ensuring that only trusted sources are used, and reducing the risk of data corruption.
-
-### About *CAS*
-
-[*The Codenotary Community Attestation Service (CAS)*](https://cas.codenotary.com) is integrated with AlmaLinux to provide a chain of trust and traceability. Each stage of the build process goes through authentication and notarization, and build artifacts are only stored if they pass authentication via *CAS*. 
-
-A user can use *CAS* to authenticate a downloaded package or its cryptographic hash (SHA256 digest) to verify if it is notarized and trusted.
-
-### Working with *SBOM* 
-
-To access the *CAS* information for AlmaLinux you need an internet connection as the *CAS* records are stored in the AlmaLinux `git` repository. You also need to install the *CAS* tool package. 
-
-Finally, to be able to inspect a desired package for *SBOM* information, you need the package itself (`.rpm` binary). Alternatively, you can use the package `build-id` or `rpm-package-hash` found on the [AlmaLinux build](https://build.almalinux.org/) page.
-
-### Supported releases
-
- * AlmaLinux 8.7
- * AlmaLinux 9.1
- 
----
-
-&nbsp;
-
-## Install guide
-
-### Software Prerequisites
-
-To just query the *SBOM* records (versus creating them) one needs only to install the above mentioned `cas` tool. *The CAS* organization provides it in form of source and binary package(s) via their [github](https://github.com/codenotary/cas) and official web service
-
-In the [README.md](https://github.com/codenotary/cas/blob/master/README.md) you can find a cross-platform installation method, presented below.
+[AlmaLinux Build System](https://github.com/AlmaLinux/build-system) has implemented *SBOM* into its pipeline for security purposes, such as tracing the build process, ensuring that only trusted sources are used, and reducing the risk of data corruption. 
 
-&nbsp;
+## About immudb
 
-### Installation 
+The Codenotary [immudb](https://docs.immudb.io/master/immudb.html#what-is-immudb) database is integrated with AlmaLinux to ensure data integrity and provide a chain of trust and traceability. Each stage of the build process goes through authentication and notarization. 
 
-#### Official *CAS* installation method (*Linux®*, *macOS*)
+AlmaLinux OS Team has developed SBOM data management [utility](https://github.com/AlmaLinux/alma-sbom) that generates SBOM records and allows users to track a package to verify if it is notarized and trusted.
```

Comment:
I suggest: `... has developed [alma-sbom](https://github.com/AlmaLinux/alma-sbom), the SBOM data management utility for AlmaLinux, that generates ...`